cassandra/policies.py

```diff
@@ -556,8 +556,13 @@ def __init__(self, base_delay, max_delay, max_attempts=64):
 
     def new_schedule(self):
         i = 0
+        delay = self.base_delay
         while self.max_attempts is None or i < self.max_attempts:
-            yield min(self.base_delay * (2 ** i), self.max_delay)
```

Comment:
self.base_delay * (2 ** i) may throw OverflowError when self.base_delay is float and i>2000